packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -315,16 +359,53 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
         finalizedWithdrawals[withdrawalHash] = true;
 
         // Set the l2Sender so contracts know who triggered this withdrawal on L2.
```

Comment:
The gas paying token must not be a double entrypoint token: https://stermi.xyz/blog/ethernaut-challenge-24-solution-double-entry-point

Reply:
https://github.com/ethereum-optimism/specs/pull/166